src/status_im/browser/permissions.cljs

```diff
@@ -21,6 +23,31 @@
                                                 :description (i18n/label :t/allowing-authorizes-this-dapp)
                                                 :icon        :main-icons/wallet}})
 
+(defn switch-ethereum-chain-permission? [requested-permission]
+  (string/starts-with? requested-permission constants/dapp-permission-switch-ethereum-chain-prefix))
+
+(defn add-ethereum-chain-permission? [requested-permission]
+  (string/starts-with? requested-permission constants/dapp-permission-add-ethereum-chain-prefix))
+
+(defn get-supported-permissions [requested-permission params]
+  (cond
+    (switch-ethereum-chain-permission? requested-permission)
```

Comment:
- because it's a `dynamic` [permission](https://github.com/qfrank/status-react/blob/d4ecbd6183ed67192e56e9953d8efca5b197cb44/src/status_im/browser/core.cljs#L374), the `chainId` is a variable 
- because i want to reuse [permission panel](https://github.com/qfrank/status-react/blob/d4ecbd6183ed67192e56e9953d8efca5b197cb44/src/status_im/ui/screens/browser/permissions/views.cljs#L45)